modules.d/90overlay-root/module-setup.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+depends() {
+    # We do not depend on any modules - just some root
+    return 0
```

Comment:
Please make this `return 255`, so it is only included, if requested.

Reply:
I cannot find any other dracut module that returns 255 here, so I don't understand why this module should do it.

Reply:
Oops.. my error... I meant in `check()`